borrowings/models.py

```diff
@@ -1,3 +1,28 @@
+from django.conf import settings
 from django.db import models
 
-# Create your models here.
+from books.models import Book
+
+
+class Borrowing(models.Model):
+    borrow_date = models.DateField()
+    expected_return_date = models.DateField()
+    actual_return_date = models.DateField(null=True, blank=True)
+
+    book = models.ForeignKey(
+        Book,
+        on_delete=models.CASCADE,
+        related_name="borrowings"
+    )
```

Comment:
```suggestion
    book = models.ForeignKey(
        Book,
        on_delete=models.CASCADE,
        related_name="borrowings",
    )
```

To avoid future conflicts between branches, it is better to put a comma at the end everywhere in such places.